File: decine_cpp/src/FestivalPremio.cpp
#include "FestivalPremio.h"

namespace decine {

	FestivalPremio::FestivalPremio(const std::string &nombre) {
		this->nombre = nombre;
	}

	std::string FestivalPremio::getNombre() {
		return this->nombre;
	}
}

File: decine_java/src/decine/FestivalPremio.java
/*  
 *  Fundamentos de Ingenieria del Software
 *  Grado en Ingeniería Informática
 * 
 *  2014 © Copyleft - All Wrongs Reserved
 *
 *  Ernesto Serrano <info@ernesto.es>
 *  
 *  
 * 
 */
package decine;

class FestivalPremio {
    private String nombre;
    
    FestivalPremio(String nombre){
        this.nombre = nombre;
    }
    
    protected String getNombre() {
        return this.nombre;
    }
}
